sev-types/src/lib.rs

```diff
@@ -16,9 +16,22 @@
 //! the SEV API specification document version; the specification publication
 //! date; the link to the SEV API specification.
 
-#![no_std]
 #![deny(missing_docs)]
 #![deny(clippy::all)]
+#![allow(missing_docs)]
```

Comment:
Let's remove this `allow` attribute, because we want to make sure the entire crate is documented. Putting this `allow` statement here would turn off the automatic check for associated doc comments. If you're unsure of what a structure represents, the AMD SEV specification is linked in the comment at the top of `lib.rs`